src/main/kotlin/no/nav/familie/ks/sak/kjerne/tilbakekreving/FagsystemsbehandlingService.kt

```diff
@@ -0,0 +1,77 @@
+package no.nav.familie.ks.sak.kjerne.tilbakekreving
+
+import no.nav.familie.kontrakter.felles.tilbakekreving.Faktainfo
+import no.nav.familie.kontrakter.felles.tilbakekreving.HentFagsystemsbehandling
+import no.nav.familie.kontrakter.felles.tilbakekreving.HentFagsystemsbehandlingRequest
+import no.nav.familie.kontrakter.felles.tilbakekreving.HentFagsystemsbehandlingRespons
+import no.nav.familie.ks.sak.common.exception.Feil
+import no.nav.familie.ks.sak.kjerne.arbeidsfordeling.ArbeidsfordelingService
+import no.nav.familie.ks.sak.kjerne.behandling.BehandlingService
+import no.nav.familie.ks.sak.kjerne.behandling.domene.Behandling
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vedtak.VedtakService
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.PersonopplysningGrunnlagService
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class FagsystemsbehandlingService(
+    private val behandlingService: BehandlingService,
+    private val personopplysningGrunnlagService: PersonopplysningGrunnlagService,
+    private val arbeidsfordelingService: ArbeidsfordelingService,
+    private val vedtakService: VedtakService,
+    private val tilbakekrevingsbehandlingHentService: TilbakekrevingsbehandlingHentService,
+    private val kafkaProducer: KafkaProducer
+) {
+
+    fun hentFagsystemsbehandling(request: HentFagsystemsbehandlingRequest): HentFagsystemsbehandlingRespons {
+        logger.info("Henter behandling for behandlingId=${request.eksternId}")
```

Comment:
Kanskje bytte det til å være "Hente fagsystemsbehandling" eller noe sånt, siden behandling kan misforstås med vanlig ks-behandling?

Reply:
@UyQuangNguyen Her henter vi faktisk ks-sak behandling for request.eksternId fra familie-tilbake som er egentlig behandlingId